doc/ug/dv_methodology/index.md

```diff
@@ -299,60 +299,112 @@ Once the regression tool is developed, it will provide a way to enable these cap
+* **Toggle Coverage**: This metric measures every logic bit to see if it transitions from 1 &rarr; 0 and 0 &rarr; 1.
   It is very difficult, and not particularly useful to achieve 100% toggle coverage across a design.
   Instead, we focus on closing toggle coverage only on the IO ports of the DUT and IO ports of pre-verified IPs within the DUT.
-*  **FSM state Coverage**: This metric measures which finite state machine states were executed during the course of a simulation.
-*  **FSM transition Coverage**: This metric measures which arcs were simulated for each finite state machine in the design.
-*  **Conditional Coverage**: This metric tracks all combinations of conditional expressions simulated.
-*  **Branch Coverage**: This metric is similar to line coverage, but not quite the same.
+* **FSM state Coverage**: This metric measures which finite state machine states were executed during a simulation.
+* **FSM transition Coverage**: This metric measures which arcs were simulated for each finite state machine in the design.
+* **Conditional Coverage**: This metric tracks all combinations of conditional expressions simulated.
+* **Branch Coverage**: This metric is similar to line coverage, but not quite the same.
   It tracks the flow of simulation (e.g. if/else blocks) as well as conditional expressions.
   Note that un-hit FSM state/transition coverage almost always shows up as un-hit branch coverage as well.
 
-### Functional Coverage
+Code coverage is sometimes referred to as implicit coverage as it is generated based on the code and takes no additional effort to implement.
 
-Unlike code coverage, functional coverage requires the designer and/or DV engineer to write additional cover points and cover groups.
-These are more complex constructs that capture whether signals (that reflect the current state of the design) have met a set of values, or a sequence of values that are interesting / corner cases.
-These constructs also allow us to capture whether multiple scenarios have occurred simultaneously through crosses, which is something the code coverage alone cannot provide.
-They provide us the ability to gauge to what extent is our stimulus through constrained random tests covering the test intent, for all tests in the testplan.
+### Functional Coverage
 
-These constructs are typically encapsulated in a class and are sometimes referred to as 'functional coverage model'.
-They are sampled in 'reactive' components of the testbench, such as monitors and / or the scoreboards.
+Unlike code coverage, functional coverage requires the designer and/or DV engineer to write additional cover points and covergroups.
+For this reason functional coverage is sometimes referred to as explicit coverage.
+These are more complex constructs that capture whether signals (that reflect the current state of the design) have met a set or a sequence of values that are interesting (often called corner cases).
```

Comment:
I realise that you've already reworded this once after a comment from me but, looking at the result, I think it can be shortened to "have met an interesting set or a sequence of values (often called corner cases)". (Sorry!)

Reply:
you are forgiven- and the text is updated